.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
It seems we don't have this leading `#` and the trailing `#` in other license blocks that use `#` as comment mark, should we keep it consistent?

Other than that, LGTM.